app/models/measure_condition_classification.rb

```diff
@@ -0,0 +1,46 @@
+class MeasureConditionClassification
+  DOCUMENT_CLASS = 'document'.freeze
+  EXEMPTION_CLASS = 'exemption'.freeze
+  THRESHOLD_CLASS = 'threshold'.freeze
+  NEGATIVE_CLASS = 'negative'.freeze
+  UNKNOWN_CLASS = 'unknown'.freeze
```

Comment:
what about using the magic comment at the beginning:
`# frozen_string_literal: true`

to avoid freezing every single string?

Reply:
I don't think we should start doing that unless we decide to do it more frequently. This would add a mix of patterns in the application which we have way too much of already